index.html

```diff
@@ -1459,6 +1461,9 @@ <h6 id="webdriver-bidi-command-permissions-setPermission">
                   </li>
                   <li>Let |state| be the value of the `state` field of |command parameters|.
                   </li>
+                  <li>Let |user context id| be the value of the `userContext` field of |command
```

Comment:
Trying to understand what a "user context" looks like... in https://w3c.github.io/webdriver-bidi/#user-context, it seems like a data structure that has an ID, has a storage partition, and has a collection of traversables? Is that right? Or is it strictly just a unique identifier and the rest happens by magic?

Reply:
The user context roughly corresponds to the user agent of the infra spec definition (except that the WebDriver can identify them). User context IDs are unique, except for the default user context (identified as "default"). In implementations, a user context is an incognito window in Chrome or a container in Firefox.

Reply:
@miketaylr is there anything else I could try to clarify?

Reply:
Sorry, closing was not intended.